Wikipedia/Code/ExploreViewController.swift

```diff
@@ -1075,6 +1075,89 @@ extension ExploreViewController {
         
         return true
     }
+    
+    private var isBeforeAssignmentEndDate: Bool {
+            var dateComponents = DateComponents()
+            dateComponents.year = 2025
+            dateComponents.month = 12
+            dateComponents.day = 31
+            guard let endDate = Calendar.current.date(from: dateComponents) else {
+                return false
+            }
+
+            return endDate >= Date()
+        }
+    
+    func presentArticleTabsAnnouncement() {
+        guard isBeforeAssignmentEndDate,
+              !WMFArticleTabsDataController.shared.hasSeenFeatureAnnouncement
+        else { return }
```

Comment:
Now you'll need to check against the ABC test assigned, since #5374 was merged. You can use shouldShowMoreDynamicTabs for that.